ui/vendor/_directional.scss

```diff
@@ -0,0 +1,63 @@
+// directional-scss | Author: Tyson Matanich (http://matanich.com), 2013 | License: MIT
+$dir: ltr !default;
+
+// Default $dir if not valid
+@if $dir != ltr and $dir != rtl {
+    $dir: ltr;
+}
+
+@function if-ltr($if, $else: null) {
+  @if $dir != rtl {
+    @return $if;
+  }
+  @else {
+    @return $else;
+  }
+}
+
+@function if-rtl($if, $else: null) {
+  @return if-ltr($else, $if);
+}
+
+$left: if-ltr(left, right);
+$right: if-ltr(right, left);
```

Comment:
hi @mattgoldspink you are bring many valid points, the switch you did initially would be least disruptive to SLDS current version, but I imagine that even from rtl languages, building an app that is `rtl` only, it would feel confusing to type `-left` when they want `right` and `-right` when they want `left`... From what it seems SLDS was not build to be non-rtl compatible.

Take a look at Atomic (https://acss.io/guides/syntax.html) that was built with `rtl` support in mind since the beginning.